libs/components/theme/src/lib/styles/_buttons.scss

```diff
@@ -75,6 +75,14 @@ a.sky-btn {
   border-color: transparent;
 }
 
+.sky-btn + .sky-btn {
```

Comment:
This has the effect of changing styles for _all_ buttons placed next to each other, regardless of whether they appear in a modal footer.

To avoid the use of `::ng-deep` and making the rules too broad, I suggest turning off view encapsulation for the modal footer component, then moving these styles into that component's SCSS file. You may also want to reconsider the rule that moves link buttons to the left by 12px. Also note I removed the mixins for modern theme and just use plain CSS selectors for those; this is required because the mixin will generate CSS that is incompatible with Firefox when view encapsulation is off.

```
import { Component, ViewEncapsulation } from '@angular/core';

/**
 * Specifies content to display in the modal's footer.
 */
@Component({
  selector: 'sky-modal-footer',
  templateUrl: './modal-footer.component.html',
  styleUrls: ['./modal-footer.component.scss'],
  encapsulation: ViewEncapsulation.None
})
export class SkyModalFooterComponent {}
```

```
@use 'libs/components/theme/src/lib/styles/mixins' as mixins;
@use 'libs/components/theme/src/lib/styles/variables' as *;

.sky-modal-footer-container {
  background-color: #fff;
  @include mixins.sky-border(light, top);
  /* Offset the margin-left of the link */
  .sky-btn-link:first-child {
    margin-left: -12px;
  }
  
  .sky-btn + .sky-btn {
    margin-left: var(--sky-adjacent-btn-margin);
  }

  .sky-btn + .sky-btn-link {
    margin-left: var(--sky-adjacent-btn-btn-link-margin);
  }
}

.sky-theme-modern {
  .sky-modal-footer-container {
    border-top: none;
    padding: $sky-theme-modern-space-lg $sky-theme-modern-space-xl;
  }
}

.sky-theme-modern-dark {
  .sky-modal-footer-container {
    background-color: transparent;
  }
}

```